teacher.rb

```diff
@@ -0,0 +1,14 @@
+# teacher.rb
+
+require_relative 'person'
+
+class Teacher < Person
+  def initialize(id, age, specialization, name = 'Unknown', parent_permission: true)
+    super(id, age, name, parent_permission)
```

Comment:
- Hey, as I've stated in my previous comment,  Please, kindly remove the id from the initialize and super.